### PR TITLE
fix faces issues

### PR DESCRIPTION
**Pull request by marcelklehr**
Signed-off-by: Marcel Klehr <mklehr@gmx.net>

### CODE REVIEW

**Review by artonge — APPROVED** [review]

File: src/views/FaceContent.vue [hunk 1]
```diff
@@ -289,7 +289,7 @@ export default {
 				path: file.filename,
 				list: this.faceFileIds.map(fileId => ({
 					...this.files[fileId],
-					basename: this.files[fileId].basename.split('-', 2)[1],
+					basename: this.files[fileId].basename.substring(this.files[fileId].basename.indexOf('-') + 1),
```
Comment (skjnldsv) [inline src/views/FaceContent.vue:292]: You can have cleaner code
```js
'1561-ben-wicks-RSG6wPidUW0-unsplash.jpg'.split('-').slice(1).join('-')
> 'ben-wicks-RSG6wPidUW0-unsplash.jpg'
```

Comment (skjnldsv) [inline src/views/FaceContent.vue:292]:
```suggestion
					basename: this.files[fileId].basename.split('-').slice(1).join('-'),
```

**Review by skjnldsv — APPROVED** [review]

File: src/views/FaceContent.vue (hunk 1 above)
Comment (skjnldsv) [reply src/views/FaceContent.vue:292]:
```suggestion
					basename: this.files[fileId].basename.split('-').slice(1).join('-'),
```

**Comment by marcelklehr** [comment]
Can't merge in this state apparenly :/

**Pull request merged**